config/webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path')
+const configPaths = require('./paths.json')
+
+const OUTPUT = process.env.OUTPUT || 'public'
+const outputPath = configPaths.output.js[OUTPUT]
+
+console.log(`Webpack will build output to ${outputPath}`)
+
+const config = {
+  mode: 'production',
```

Comment:
This might have been discussed before but is there a reason we don't link this to an environment variable to toggle `production` and `development`?  [This](https://medium.com/webpack/webpack-4-mode-and-optimization-5423a6bc597a) makes it sound like there could be some cool advantages to using `development` where necessary? 

Completely not a blocking comment, I'm just curious!

Reply:
I *think* the main reason we're not flipping the mode is that we're in a halfway house between Webpack and Gulp.

We'd want to move the Sass bundling to webpack, and rely on webpack only which'd make it less painful to take advantage of development mode.

Reply:
@hannalaakso: we set it to production since in this project we're using Webpack to provide production-ready JavaScript for others to use in their project. We've setup devtool to generate  source maps that covers the debugging advantages presented in that article.